torch/ao/quantization/backend_config/_common_operator_config_utils.py

```diff
@@ -393,21 +394,43 @@ def _get_default_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPat
     )
     return configs
 
+def _add_fixed_qparams_to_dtype_configs(
+    dtype_configs: List[DTypeConfig],
+    observer: FixedQParamsObserver,
+) -> List[DTypeConfig]:
+    """
+    Return a copy of the list of DTypeConfigs where activations are subject to the fixed qparams
+    constraints specified in the observer.
+
+    If `scale_min_lower_bound` or `scale_max_upper_bound` is specified in the activations, throw an
+    exception since these settings are incompatible with fixed qparams ops.
+    """
+    fixed_scale = observer.scale
+    fixed_zero_point = observer.zero_point
+    new_dtype_configs = []
+    for dtype_config in dtype_configs:
+        dc = copy.deepcopy(dtype_config)
+        for dtype_with_constraints in [dc.input_dtype_with_constraints, dc.output_dtype_with_constraints]:
+            dtype_with_constraints.fixed_scale = float(fixed_scale)
+            dtype_with_constraints.fixed_zero_point = int(fixed_zero_point)
+            if dtype_with_constraints.scale_min_lower_bound is not None:
+                raise ValueError("scale_min_lower_bound is invalid for fixed qparams ops: %s" % dtype_config)
+            if dtype_with_constraints.scale_max_upper_bound is not None:
+                raise ValueError("scale_max_upper_bound is invalid for fixed qparams ops: %s" % dtype_config)
+        new_dtype_configs.append(dc)
+    return new_dtype_configs
+
 def _get_fixed_qparams_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPatternConfig]:
     fixed_qparams_op_configs = []
-    for fixed_qparam_op, output_observer in _FIXED_QPARAMS_OP_TO_OBSERVER.items():
+    for fixed_qparam_op, obs_ctr in _FIXED_QPARAMS_OP_TO_OBSERVER.items():
```

Comment:
it's surprising to depend on qconfig here, since qconfig and backend_config are supposed to be configuring different things.

How about something like:
1. there is a mapping of PyTorch op to fixed qparams (not related to qconfig or backend_config)
2. there is a mapping from (1) to qconfig
3. there is a mapping from (1) to backend_config

Reply:
Actually `_FIXED_QPARAMS_OP_TO_OBSERVER` is not part of qconfig, so what you're describing is already the current behavior. We do have a future BE task planned to move this map somewhere else (outside of qconfig.py), but that's separate from this PR. I can add a TODO here

Reply:
observer objects are intended to be used in `QConfig` only, so the current code is using qconfig related objects indirectly.  Can we fix this in the same stack as this PR?  It's confusing to reuse these abstractions outside of qconfig.

Reply:
I see. Ok, what I can do then is make a map from the ops to the expected qparams, then derive the observers based on this map.